canonical/canonical.com
# WD-1490 Add contact number for Korea.

## Done

Add contact number for Korea.
Update numbers with ubuntu.com/contact-us

## QA

Check phone number is correct in the jira task
https://warthogs.atlassian.net/browse/WD-1490?atlOrigin=eyJpIjoiOWViM2ExNmJhZTNhNGIzNWJjMjk4YTBjZGM5MDYxNjQiLCJwIjoiaiJ9


## Issue / Card

Fixes #

## Screenshots

[if relevant, include a screenshot]
